scripts/start.js

```diff
@@ -347,6 +357,10 @@ function getUserInfo () {
     defaultInput: DEFAULT_DOMAIN
   });
 
+  var userId = readlineSync.question(`oVirt user id (${DEFAULT_UUID}), can be found there ${engineUrl}api/users/ : `, {
```

Comment:
@jniederm @mareklibra but we can't, I tried again on brq-dev call api/users under `ovirt02`, and still I get empty object `{}`.